presto-docs/src/main/sphinx/admin/verifier.rst

```diff
@@ -166,19 +166,70 @@ queries.
     * Checks the nullity of control sum and test sum.
     * If either control mean or test mean very close 0, checks if both are close to 0.
     * Checks the relative error between control sum and test sum.
+* **VARCHAR Columns**
+    * For ``VARCHAR`` columns, a simple checksum column is generated for verification using :func:`!checksum`.
+    * If ``validate-string-as-double`` is set, 6 additional columns below are generated. If ``NULL`` counts
+      are equal before and after casting all values to ``DOUBLE``, apply the aforementioned floating point validation.
+      Otherwise, check if the simple checksum matches.
+        * Count of the ``NULL`` values
+        * Count of the ``NULL`` values after all values are casted to ``DOUBLE``
+        * After casting all values to ``DOUBLE``
+            * Sum of the finite values
+            * ``NAN`` count of the values
+            * Positive infinity count of the values
+            * Negative infinity count of the values
 * **Array Columns**
-    * 2 columns are generated for verification:
-       * Sum of the cardinality
-       * Array checksum
-    * For an array column ``arr`` of type ``array(E)``:
-       * If ``E`` is not orderable, array checksum is ``checksum(arr)``.
-       * If ``E`` is orderable, array checksum ``coalesce(checksum(try(array_sort(arr))), checksum(arr))``.
+    * For an array column ``arr`` of type ``array(E)``, 3 columns are generated for verification:
+        * Sum of the cardinality
+        * Checksum of the cardinality
+        * Array checksum
+            * If ``E`` is not orderable, array checksum is ``checksum(arr)``.
+            * If ``E`` is orderable, array checksum is ``coalesce(checksum(try(array_sort(arr))), checksum(arr))``.
+    * If ``use-error-margin-for-floating-point-arrays`` is set and E is ``DOUBLE`` or ``REAL``,
+      6 columns below are generated instead. Check if the sum of the cardinality matches and if the checksum of the cardinality
+      matches. And apply the aforementioned floating point validation to the rest of results.
+        * Sum of the cardinality
+        * Checksum of the cardinality
+        * Sum of the finite elements of all array values
+        * Count of the ``NAN`` elements of all array values
+        * Count of the positive infinity elements of all array values
+        * Count of the negative infinity elements of all array values
+    * If ``validate-string-as-double`` is set and E is ``VARCHAR``, 9 columns below are generated instead.
+      Check if the sum and the checksum of the cardinality match. If ``NULL`` counts are equal before and after
+      casting all array elements to ``DOUBLE``, apply the aforementioned floating point validation. Otherwise, check
+      if the array checksum matches.
+        * Sum of the cardinality
```

Comment:
Error in local doc build: 

`/Users/steveburnett/Documents/GitHub/presto/presto-docs/src/main/sphinx/admin/verifier.rst:201: ERROR: Unexpected indentation.`

Suggest unindenting this list, and any nested list within it, and do a local doc build until these similar errors do not appear. See [Building the Documentation](https://github.com/prestodb/presto/tree/master/presto-docs#building-the-documentation) for more information.